plutus-scripts/PlutusScripts/Batch6/BLS12_381_MultiScalarMul/G1/Common.hs

```diff
@@ -0,0 +1,448 @@
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE RecordWildCards #-}
+{-# LANGUAGE TemplateHaskell #-}
+
+{- | Simple end-to-end tests for the Plutus Core `BLS12_381_G1_MultiScalarMul`
+builtin.  Some of these are adapted from `plutus-conformance` tests: see the
+numbers in parentheses.  There are no inputs which would cause
+`BLS12_381_G1_MultiScalarMul` to fail, so we only provide succeeding tests.
+-}
+module PlutusScripts.Batch6.BLS12_381_MultiScalarMul.G1.Common (
+  mkSucceedingMultiScalarMulPolicy1,
+  succeedingMultiScalarMulParams1,
+  mkSucceedingMultiScalarMulPolicy2,
+  succeedingMultiScalarMulParams2,
```

Comment:
This naming scheme (suffixes 1, 2) doesn't reveal the difference in their nature. I understand that we can't always express such difference in a name itself, so maybe you can leave a comment explaining why 2 instances needed and how they differ?
P.S.: It has crossed my mind right after writing the question above: do suffixes 1 and 2 mean G1 and G2 ?